docs/spec/v1beta1/receiver.md

```diff
@@ -247,6 +247,10 @@ spec:
       name: webapp
 ```
 
+Note that you have to fill in the generated token as the secret key when creating the Nexus Webhook Capability.
+See [nexus documentation](https://help.sonatype.com/repomanager3/webhooks/enabling-a-repository-webhook-capability)
```

Comment:
move this link on Nexus Webhook Capability please.

Reply:
I should remove it? Or move it somewhere else?

Reply:
`[Nexus Webhook Capability](https://help.sonatype.com/repomanager3/webhooks/enabling-a-repository-webhook-capability)`